examples/CMakeLists.txt

```diff
@@ -9,25 +9,29 @@ set(EXAMPLES
 set(EXAMPLES_OPENMP threadsafe1d1 threadsafe2d2f)
 set(EXAMPLES_C guru1d1c simple1d1c simple1d1cf)
 
+find_library(MATH_LIBRARY m)
+
 foreach(EXAMPLE ${EXAMPLES})
   add_executable(${EXAMPLE} ${EXAMPLE}.cpp)
-  target_compile_features(${EXAMPLE} PRIVATE cxx_std_17)
+  target_compile_features(${EXAMPLE} PRIVATE cxx_std_14)
   target_link_libraries(${EXAMPLE} PRIVATE finufft)
   enable_asan(${EXAMPLE})
```

Comment:
might be good to do, so in case this is used standalone a not existent function is not called:
```
  if(CMAKE_PROJECT_NAME STREQUAL "FINUFFT")
    enable_asan(${EXAMPLE})
  endif()
```